System/Console/ANSI/Types.hs

```diff
@@ -14,7 +18,7 @@ import Data.Ix (Ix)
 
 import Data.Colour (Colour)
 
--- | ANSI colors: come in various intensities, which are controlled by
+-- | ANSI 3/4-bit "candy" colors: come in various intensities, which are controlled by
```

Comment:
It's not obvious what "candy" means here, could you clarify?

Reply:
It's a term I saw here and there, for example in https://stackoverflow.com/questions/15682537/ansi-color-specific-rgb-sequence-bash

I can remove it, as it doesn't seem to be official and use "4-bit colors" instead

Reply:
Yeah, you can remove it or explain it — either way is fine with me.